concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -667,9 +624,7 @@ jobs:
           - get: concourse-tfstate
           - get: bosh-tfstate
           - get: cf-tfstate
-          - get: metrics-cert
-          - get: logsearch-cert
-            passed: ['generate-cf-secrets']
+          - get: cf-certs
```

Comment:
This should probably be pinned to `generate-cf-secrets` to ensure it gets the up-to-date version.